.travis.yml

```diff
@@ -44,6 +50,9 @@ matrix:
       env: BUILD_TYPE=Release VERBOSE=1 CXX_FLAGS=-std=c++11
       if: type != pull_request
 
+before_install:
```

Comment:
This looks like it would apply to all entries in the build matrix. ( Correct me if I am wrong ). It would be preferable to contain the platformio changes only to platformio builds

Reply:
It would apply to all builds.  I can look at applying only to PIO.  I am not sure why I need it.  The other builds execute scripts out of the ci directory without the explicit +x.  I would prefer to do it that way but my Travis CI skills are not advanced.

Any ideas there?

Reply:
I just thing that if we are adding PIO specific builds then all changes should be self-contained to PIO only. I believe you can just move this into the PIO section in the travis.yml file